src/node/rpc/member_frontend.h

```diff
@@ -330,6 +382,162 @@ namespace ccf
       return tx_modules->remove(name);
     }
 
+    void remove_jwt_keys(kv::Tx& tx, std::string issuer)
+    {
+      auto keys = tx.get_view(this->network.jwt_public_signing_keys);
+      auto key_issuer =
+        tx.get_view(this->network.jwt_public_signing_key_issuer);
+
+      key_issuer->foreach(
+        [&issuer, &keys, &key_issuer](const auto& k, const auto& v) {
+          if (v == issuer)
+          {
+            keys->remove(k);
+            key_issuer->remove(k);
+          }
+          return true;
+        });
+    }
+
+    bool set_jwt_public_signing_keys(
+      kv::Tx& tx,
+      ObjectId proposal_id,
+      std::string issuer,
+      const JsonWebKeySet& jwks)
+    {
+      auto issuers = tx.get_view(this->network.jwt_issuers);
+      auto keys = tx.get_view(this->network.jwt_public_signing_keys);
+      auto key_issuer =
+        tx.get_view(this->network.jwt_public_signing_key_issuer);
```

Comment:
FYI, you can `auto [view1, view2, view3] = tx.get_view(map1, map2, map3);` in one line.

Reply:
To be honest, I think that's quite weird and should be removed :/

Reply:
Is it? Seems like a nice way to declare all your views in one place. Anyway, either is fine.